### PR TITLE
Adding explanatory text to emission-pledges.md

### PR DESCRIPTION
**Pull request by willemfrishert**
Task of #14: Add English texts to the published infographics

### Preview Build

Current preview build is available at https://en-facts-on-climate--preview-emission-pledges-hjb4v3u0.web.app/infographics/emission-pledges

### What does this PR do?

- bulk-translated the Czech text to English using [DeepL](https://www.deepl.com) for emission-pledges.md
- Use [Grammarly](https://www.grammarly.com/) to proofread the texts.

### Commented out links:
- Commented out preview-box (using Jekyll comments) as these are (not yet) available in English
  -  [slug="uhlikova-neutralita - https://faktaoklimatu.cz/explainery/uhlikova-neutralita](https://faktaoklimatu.cz/explainery/uhlikova-neutralita)

- Commented out link in text (using html comments) as these are (not yet) available in English:
  - [explainery/uhlikova-neutralita - https://faktaoklimatu.cz/explainery/uhlikova-neutralita](https://faktaoklimatu.cz/explainery/uhlikova-neutralita)
  - [studie/2021-reserse-zavazky-statu - https://faktaoklimatu.cz/studie/2021-reserse-zavazky-statu](https://faktaoklimatu.cz/studie/2021-reserse-zavazky-statu)

### CODE REVIEW

File: collections/_infographics/policies/emission-pledges.md [hunk 1]
```diff
@@ -12,4 +12,34 @@ data-orig:
   - [ "Our Jupyter notebook", "https://github.com/faktaoklimatu/data-analysis/blob/master/notebooks/net-zero-pledges.ipynb" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emisni-zavazky" %}
+## How to read an infographic?
```
Comment (tjarnikova) [inline collections/_infographics/policies/emission-pledges.md]: This is the direct translation from Czech but the phrasing doesn't work in English. Let's write "How to read this infographic" or "How to read this infographic:"

File: collections/_infographics/policies/emission-pledges.md [hunk 2]
```diff
@@ -12,4 +12,34 @@ data-orig:
...
+{% comment %}
+{% include preview-box.html
+    title="What is carbon neutrality?"
+    text="We explain exactly what the zero carbon commitment means and how the term can be confusing."
+    slug="uhlikova-neutralita"
+%}
+{% endcomment %}
+This infographic shows the share of carbon-neutral countries in global CO<sub>2</sub> emissions in 2018. Excluding the Land Use, Land Use Change and Forestry sector (*LULUCF), all CO<sub>2</sub> emissions, including international maritime and aviation transport. **As of March 2022, 87% of global emissions come from countries moving towards carbon neutrality**, with 131 countries total.
```
Comment (tjarnikova) [inline collections/_infographics/policies/emission-pledges.md]: small: no need for a * in (*LULUCF); not common in English acronyms in my experience.

Comment (tjarnikova) [reply collections/_infographics/policies/emission-pledges.md]: the 131 countries total is ambiguous here. Rephrase as:

**As of March 2022, the 131 countries that are moving towards carbon neutrality are responsible for 87% of global emissions**.

File: collections/_infographics/policies/emission-pledges.md [hunk 3]
```diff
@@ -12,4 +12,34 @@ data-orig:
...
+
+Countries that are moving towards carbon neutrality are those that have commitments: fulfilled (e.g., Bhutan), set in law, set in a policy document, in the form of a declaration/commitment, or at the proposal/discussion stage. If we exclude countries with commitments in the draft or discussion phase, there would be a 3% reduction in covered emissions to 84% of global emissions.
```
Comment (tjarnikova) [inline collections/_infographics/policies/emission-pledges.md]: slight rephrase for clarity:

Countries that are moving towards carbon neutrality are those whose commitments are either fulfilled (e.g., Bhutan), set in law, set in a policy document, in the form of a declaration/commitment, or at the proposal/discussion stage.

File: collections/_infographics/policies/emission-pledges.md [hunk 4]
```diff
@@ -12,4 +12,34 @@ data-orig:
...
+
+Some countries claim to be climate neutral (*climate neutrality, net-zero emissions*) and some to carbon neutrality (*carbon neutrality, net-zero carbon*). However, we do not distinguish between them in the infographic and use the term carbon neutrality for all commitments. While the former refers to all greenhouse gases, the latter refers "only" to CO<sub>2</sub>. I.e., if a country commits to climate neutrality, this commitment includes carbon neutrality. <!--For more information on carbon neutrality in general, see our [explainer](/explainers/carbon-neutrality).--> Countries worldwide must achieve carbon neutrality by around 2050 to keep global warming below 1.5°C. Commitments towards later dates are insufficient to meet this goal.
```
Comment (tjarnikova) [inline collections/_infographics/policies/emission-pledges.md]: rephrase: "Some countries claim to be climate neutral (*climate neutrality, net-zero emissions*) and some to carbon neutrality (*carbon neutrality, net-zero carbon*).  However, we do not distinguish between them in the infographic and use the term carbon neutrality for all commitments."

as "Some countries aim for climate neutrality (*climate neutrality, net-zero emissions*) and some for carbon neutrality (*carbon neutrality, net-zero carbon*). We do not distinguish between these two terms in this infographic and use the term carbon neutrality for all commitments."

File: collections/_infographics/policies/emission-pledges.md [hunk 5]
```diff
@@ -12,4 +12,34 @@ data-orig:
...
+2. **USA**: 13.7% share, 2050,
+3. **EU-27**, 8.2% share, 2050
+4. **India**, 6.7% share, 2070,
+5. **Russia**, 4.8% share, 2060.
+<!--
+We summarise the commitments of other states in the research paper [Emission commitments of states](/studie/2021-reserse-zavazky-statu).
+-->
+## Sources
+
+* The source of carbon neutrality terms is the [Net Zero Tracker](https://zerotracker.net) project.
```
Comment (tjarnikova) [inline collections/_infographics/policies/emission-pledges.md]: I would change "Sources" to "References"; more standard.

Comment (tjarnikova) [reply collections/_infographics/policies/emission-pledges.md]: change "The source of carbon neutrality terms is the [Net Zero Tracker](https://zerotracker.net) project." to "The reference for the carbon neutrality terms is the [Net Zero Tracker](https://zerotracker.net) project."

File: collections/_infographics/policies/emission-pledges.md [hunk 6]
```diff
@@ -12,4 +12,34 @@ data-orig:
...
+* The source of the emissions database is the [EDGAR](https://edgar.jrc.ec.europa.eu/) database (*Emission Database for Global Atmospheric Research*) operated by the European Environment Agency.
```
Comment (tjarnikova) [inline collections/_infographics/policies/emission-pledges.md]: this one is optional, would change to "The reference for the emissions database is the [EDGAR](https://edgar.jrc.ec.europa.eu/) database (*Emission Database for Global Atmospheric Research*) operated by the European Environment Agency."

**Review by tjarnikova — CHANGES_REQUESTED** [review]

**Comment by tjarnikova** [comment]
@katkolouchova @willemfrishert  ahoj! the infographic needs a slight text adjustment here - the placement of 131 doesn't work with English sentence structure. Change the opening sentence to "The 131 countries which **have** pledged to become carbon neutral account for 87% of global CO2 emissions", or, if you want to keep the structure, "87% of global CO2 emissions come from the 131 countries which **have** pledged to become carbon neutral".

File: collections/_infographics/policies/emission-pledges.md (hunk 4 above)
Comment (willemfrishert) [reply collections/_infographics/policies/emission-pledges.md]: Thanks for catching this. I updated the text.

File: collections/_infographics/policies/emission-pledges.md (hunk 2 above)
Comment (willemfrishert) [reply collections/_infographics/policies/emission-pledges.md]: I did not intend to keep the asterisk. Removed it. Thanks for catching this.

Comment (willemfrishert) [reply collections/_infographics/policies/emission-pledges.md]: I updated text in the paragraph as suggested.

File: collections/_infographics/policies/emission-pledges.md (hunk 5 above)
Comment (willemfrishert) [reply collections/_infographics/policies/emission-pledges.md]: I updated the word "Source" and text to use "References".

File: collections/_infographics/policies/emission-pledges.md (hunk 6 above)
Comment (willemfrishert) [reply collections/_infographics/policies/emission-pledges.md]: I ended up applying the same change here: "The source ..." -> "The reference ..."

**Comment by willemfrishert** [comment]
@katkolouchova @tjarnikova

I updated the regular text as per suggestions. I suggest we create a new ticket for so that @ katkolouchova can fix it independently from this pull request.

**Comment by katkolouch** [comment]
@tjarnikova @willemfrishert Thanks for the remark :), I have edited the subtitle accordingly.

**Comment by tjarnikova** [comment]
looks good to me!

**Review by tjarnikova — APPROVED** [review]

**Comment by tjarnikova** [comment]
Hi @mukrop - @willemfrishert have been through this page and it looks good to me, and I've merged. I hope this workflow is correct, let me know if I'm missing anything. Cheers!

**Comment by mukrop** [comment]
@tjarnikova, @willemfrishert, good job, the two of you :-).

I apologize for not being able to get to you in the past weeks. The newborn is a bit more demanding than expected (though that was to be expected :D).

**Pull request merged**